packages/react-components/react-motion-components-preview/library/src/components/Atoms/Atoms.ts

```diff
@@ -0,0 +1,21 @@
+import { AtomMotion } from '@fluentui/react-motion';
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33259&commitId=7b13ae8b219aad612576732afd97e88dee5f2059&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|6| Changed|
|Avatar Converged.badgeMask.chromium.png|12| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1171| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>